tests/reconfig/setup.md

```diff
@@ -26,7 +26,7 @@
 
  The following cluster will be sufficient:
 
-- A Kubernetes cluster with 3 nodes on GKE
+- A Kubernetes cluster with 4 nodes on GKE
```

Comment:
3 nodes were not enough for the tests with 150 namespaces. Some Pods were stuck in pending.